lnwallet/channel.go

```diff
@@ -2127,6 +2155,38 @@ func NewBreachRetribution(chanState *channeldb.OpenChannel, stateNum uint64,
 				return nil, err
 			}
 		}
+
+		// At this point, we'll check to see if we need any extra
+		// resolution data for this output.
+		resolveReq := ResolutionReq{
+			ChanPoint:      chanState.FundingOutpoint,
+			ShortChanID:    chanState.ShortChanID(),
+			Initiator:      chanState.IsInitiator,
+			FundingBlob:    chanState.CustomBlob,
+			Type:           input.TaprootRemoteCommitSpend,
```

Comment:
I think this is correct. Since we're in the `ourAmt >= dust` branch, this is the remote commitment's unencumbered `to_remote` output that goes to us. We have the `TaprootCommitmentRevoke` in the `theirAmt >= dust` branch below.
@Roasbeef can you confirm please?

Reply:
Yeah this is fine here, on the server side we're able to recognize the breach case properly.